homeassistant/components/flume/binary_sensor.py

```diff
@@ -68,6 +69,13 @@ class FlumeBinarySensorEntityDescription(
         event_rule=NOTIFICATION_HIGH_FLOW,
         icon="mdi:waves",
     ),
+    FlumeBinarySensorEntityDescription(
+        key="low_battery",
+        name="Low battery",
```

Comment:
Since entity translation has been implemented in this integration earlier (see #95350), we cannot omit the name (as it will be provided by the device class in this case).

```suggestion
```